{"uuid": "93d3f22f-c88b-44fd-86db-3bc26184559e", "children": ["38926525-2016-4e66-9fc5-ecf30352513b"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable\n", "trace": "  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\Desktop\\SW Testing\\Pytest framework\\CredKart_pytest_project 31-07-2023\\CredKart_Pytest_Project - Copy\\testCases\\conftest.py\", line 27, in setup\n    driver = webdriver.edge()\n             ^^^^^^^^^^^^^^^^\n"}, "start": 1691475463384, "stop": 1691475463384}], "start": 1691475463384, "stop": 1691475463460}